components/vf-button/vf-button.scss

```diff
@@ -16,10 +16,11 @@ $button-hover-fix: -5px -5px rgba(0, 0, 0, 0);
   --vf-button__color__background--default: var(--vf-color--grey);
   --vf-button__color__background--default-dark: var(--vf-color--grey--darkest);
   --vf-button__color__border--default-visited: var(--vf-color--grey--lightest);
-  
+
   @include set-type(text-button--1);
 
   appearance: none;
+  backface-visibility: hidden;
```

Comment:
Do we need this? It's probably not hurting anything though.

Reply:
I thought it looked 'better' when testing it in webkit browsers with this. Could be my eyes :sob: